.github/workflows/security.yml

```
@@ -0,0 +1,14 @@
+name: Security
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Perhaps it is too much to run daily. can we run this weekly ?

```suggestion
    - cron: '0 0 * * 0'
```

---

I also have a question, mostly because I don't know about this github action, what would happen if the job the next day and there is already an issue open for that ? Would it skip and create-new one ?. If it doesn't do that repetitively, I am fine with running it daily.

---

According to my own experience, if the issue is already opened, it won't create another one :)